content/en/developers/guide/unified-tagging-advanced-usage.md

```diff
@@ -0,0 +1,66 @@
+
+#### APM
+
+`env` and `service` are core tags in APM, so it is not possible to replace them with differently named tags. However, APM does allow for data to be [aggregated along more primary tags] than just `env`. Since host tags are added to traces and trace metrics, they can be used as well (e.g., availability-zone).
+
+Trace metrics also have a specific [whitelist of tags][2]. Container tags are not included.
+
+Autodiscovered tags associated with containers are added to `container_info` in span metadata.
+
+#### Logs
+Similar to APM, service is a core tag that is used to help organize logs data. Additionally, it is not possible to link from a log to the related APM service without it.
+
+Similar to metrics, however, Autodiscovered tags for the container as well as host tags for the agent are added to all logs.
+
+You also have the ability to add custom fields to your logs in-code, which can be mapped to tags or attributes downstream in a [Datadog log processing pipeline][3].
+
+## Using Standard Labels
+
+We recommend using both standard labels and environment variables. However, standard labels can be seen as an alternative to using environment variables, particularly for applications that will not benefit from using those variables in their runtime. 3rd party software like Redis, MySQL, and Nginx are a few examples. Since these services still generate infrastructure metrics and data from integration checks, it's valuable to have all of that data tagged with `env`, `service`, and `version`.
+
+If you would like to tag Kubernetes state metrics with `env`, `service`, and `version` then the standard labels offer the easiest way. The `DD` environment variables of a container cannot be used by the Agent to tag these metrics, so the labels are a more natural option.
+
+### Declaring environment as a label
+
+It's important to declare `env `as a label even if it's already defined in your agent. Configuring `env` closer to the source of the data (e.g., APM traces, logs) helps to avoid inconsistencies where the Agent `env` might be different. Making `env` part of the service's configuration guarantees a service-centric source of truth.
+
+### Using standard labels with existing Kubernetes tags annotation
+Kubernetes users can continue to use these general tags. However, using the specific labels has a few benefits:
+
+- You can directly reference them for environment variable injection via the Kubernetes downward API.
+- The service standard label can simplify the definition of service for logs.
+
+### Using standard labels for specific containers
+Since the `DD` environment variables are injected at the container level, they can differ from container to container. However, if you want to use the standard labels as well for specific containers, then you will need to use their container-specific variants:
```

Comment:
```suggestion

Since the `DD` environment variables are injected at the container level, they can differ from container to container. However, if you want to use the standard labels as well for specific containers, then you will need to use their container-specific variants:
```